opm/autodiff/FlowMainEbos.hpp

```diff
@@ -232,7 +241,24 @@ namespace Opm
         // Throws std::runtime_error if failed to create (if requested) output dir.
         void setupOutput()
         {
-            output_to_files_ = output_cout_ && param_.getDefault("output", true);
+            std::string output = param_.getDefault("output", std::string("all"));
+            static std::map<std::string, FileOutputValue> string2OutputEnum =
+                { {"none", OUTPUT_NONE },
+                  {"false", OUTPUT_LOG_ONLY },
```

Comment:
One could also argue that "false" and "none" should act the same, but I'm fine with this as it preserves existing behaviour. Your call.